client/src/components/Pages/Experience/index.js

```diff
@@ -11,34 +11,34 @@ import { experiences } from './constants';
 
 import './styles.css';
 
-const Experience = (props) => {
-  const renderExperiences = () => (
+const Experience = props => {
+  const renderExperiences = () =>
     experiences.map(item => (
       <ExperienceItem
         key={item.id}
         discovered={props[item.nameInStore]}
         {...item}
       />
-    ))
-  );
+    ));
 
   const renderContent = () => (
     <React.Fragment>
       <h3 className="experiences-section-title">Experience</h3>
       <ProgressBar percent={props.discoveryPercent} margintTop />
-      <div className="experiences-wrapper">
-        {renderExperiences()}
-      </div>
+      <div className="experiences-wrapper">{renderExperiences()}</div>
     </React.Fragment>
   );
 
   return (
     <div className="experiences-section-container">
-      {
-        props.discovered
-          ? renderContent()
-          : <EmptyView section="experience" suggestion="Describe your working experience" />
-      }
+      {props.discovered ? (
+        renderContent()
```

Comment:
Bad fix, I hope this will be fixed with the length increase 🤔

Reply:
This is not caused by the line length 😕
Seems its going to be like this because prettier thing is better 🤷‍♂️